addons/project/static/src/views/burndown_chart/burndown_chart_renderer.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module **/
+
+import { GraphRenderer, shortenLabel } from "@web/views/graph/graph_renderer";
+import { useService } from "@web/core/utils/hooks";
+
+const { onWillStart } = owl;
+
+
+export class BurndownChartRenderer extends GraphRenderer {
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+
+        onWillStart(async () => this.stageSequence = await this.fetchSequence());
+    }
+
+    async fetchSequence() {
+        const activeId = this.model.metaData.context.active_id;
+        return await this.orm
+            .webSearchRead(
+                'project.task.type',
+                [['project_ids', 'in', activeId]],
+                ["name","sequence"]
+            ).then((data) => {return data.records})
+    }
+
+    /**
+     * @returns {Object}
+     * @override
+    **/
+    getLegendOptions() {
```

Comment:
weird things happen when clicking on a legend label.
it will be easier and cleaner to change the input rather hack the output

Reply:
Removed the renderer, added the sequence sorting in the burndown_chart_model